tests/sanity/refresh_ignore_files

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+
+from pathlib import Path
+
+target_dir = Path('.')
+
+ignore_dir = target_dir / "tests" / "sanity"
+module_dir = target_dir / "plugins" / "modules"
+module_utils_dir = target_dir / "plugins" / "module_utilss"
+ignore_dir.mkdir(parents=True, exist_ok=True)
+
+for version in ["2.9", "2.10", "2.11", "2.12"]:
+    ignore_file = ignore_dir / f"ignore-{version}.txt"
+    ignore_content = ignore_file.read_text().split("\n")
+    ignore_content.append(f"tests/sanity/refresh_ignore_files shebang!skip")
+    skip_list = [
+        "compile-2.6!skip",  # Py3.8+
+        "compile-2.7!skip",  # Py3.8+
+        "compile-3.5!skip",  # Py3.8+
+        "compile-3.6!skip",  # Py3.8+
+        "compile-3.7!skip",  # Py3.8+
```

Comment:
Do we even need lines for < 3.6? We are only testing on 3.6+. I think I'd also prefer we don't skip the compile check on versions we support. I don't think we've had issues in the past with this check. I would say just remove compile from this list.

Reply:
Zuul runs all the versions of Python for sanity tests. There is no simple way to disable these checks unless we add these lines for non-supported versions. This is already been followed in `amazon.aws` repo.